tests/unit/AddStaffNotationDialog.spec.ts

```diff
@@ -13,6 +13,9 @@ Vue.use(Vuetify)
 const vuetify = new Vuetify({})
 const store = getVuexStore()
 describe('AddStaffNotationDialog', () => {
+  // Avoid vuetify warnings related to v-dialog
```

Comment:
There's actually a standard comment and location for this code in other tests. Please copy that (for consistency).